app/dashboard/views.py

```diff
@@ -6479,8 +6561,19 @@ def payout_bounty_v1(request, fulfillment_id):
     is_funder = bounty.is_funder(user.username.lower()) if user else False
 
     if not is_funder:
-        response['message'] = 'error: payout is bounty funder operation'
-        return JsonResponse(response)
+        is_owner = False
```

Comment:
is it possible to update is_funder logic with the is_owner logic ?
Also what's the diff btwn funder and owner at this point ?